assets/js/container/register-maker/register-maker.js

```diff
@@ -285,11 +297,28 @@ class RegisterMaker extends React.Component {
     if (this.context.user && this.context.user.id) {
       // todo redirect to home?
     }
+
+    this.getCountryList("de");
   }
 
-  onSubmit = (data) => {
-    console.log(data);
+  getCountryList(lang) {
+    const url = `https://restcountries.eu/rest/v2/all?fields=alpha2Code;name;translations`;
+    const langIsSupported = ["de", "es", "fr", "ja", "it", "br", "pt"].includes(lang);
+
+    axios.get(url)
```

Comment:
please use fetch instead of axios. for details look at dashbord.js or https://reactjs.org/docs/faq-ajax.html

Reply:
as i know you can send any kind of request with fetch and there is no need for an external client or package. to keep the dependecies light we should replace it with fetch. we also have a ticket to replace the existing axios calls with fetch in the hole project, issue 208.